modules/command/dh/dh-process-blockchain-events-command.js

```diff
@@ -0,0 +1,148 @@
+const Command = require('../command');
+const Utilities = require('../../Utilities');
+const Models = require('../../../models/index');
+const constants = require('../../constants');
+
+/**
+ * Repeatable command that checks whether offer is created or litigation is successfully initiated
+ */
+class DHProcessBlockchainEventsCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.config = ctx.config;
+        this.logger = ctx.logger;
+        this.commandExecutor = ctx.commandExecutor;
+        this.blockchain = ctx.blockchain;
+        this.profileService = ctx.profileService;
+        this.errorNotificationService = ctx.errorNotificationService;
+        this.dhService = ctx.dhService;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        try {
+            const events = await Models.events.findAll({
+                where: {
+                    event: {
+                        [Models.Sequelize.Op.in]: [
+                            constants.EVENT_NAME.LitigationInitiated,
+                            constants.EVENT_NAME.OfferCreated],
```

Comment:
So OfferFinalized will be put here in the future?

Reply:
Yes :-)